env/env.go

```diff
@@ -0,0 +1,49 @@
+package env
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+var (
+	configuration *Static
+	ConfigFlag    string = "DOWNDETECTOR_CONFIG"
+)
+
+type Static struct {
+	BotToken    string `json:"bot_token"`
+	BotGuild    string `json:"bot_guild"`
+	Address     string `json:"address"`
+	ChannelName string `json:"channelName"`
+	Checks      []Check
+}
+
+type Check struct {
+	Type     string `json:"type"`
+	Value    string `json:"value"`
+	Interval string `json:"interval"`
```

Comment:
Add a new field called Parameters like this:
```
type Parameters struct {
  StatusCode int
}
```
and add it like 
```
type Check struct {
  ...
  Parameters *Parameters json.... 
}
```
So if this field is passed `check.Parameters != nil`, than check for that status code instead of 200, if not then use 200.